### ISSUE TITLE
should be on gog.com

### ISSUE THREAD

**Issue by shezi** [endless-sky/endless-sky#147]
It would be awesome if this game was available on gog.com (without Steam DRM).

I can do the submission for you if you want. The guys from gog.com seem pretty cool, so it shouldn't be a big problem.

More information here: http://www.gog.com/indie

**Comment by Andrettin** [comment]
Unfortunately GOG.com is not very open-source friendly, as they say they would reject a game if it uses free assets. From their game submission FAQ (in that very link you posted):

"I used some free or borrowed assets from different game, is this a problem?"

"Yes, this is an issue for us. We're strongly focused on more original content and free or borrowed assets mean that there's a very small chance we'll accept the game."

They also say there that they don't accept Free-to-Play games (maybe they would accept one which doesn't have microtransactions, though).

**Comment by shezi** [comment]
They have a handful of completely free games in their store, so I think that shouldn't be a problem. What I guess they don't want to have in their store is microtransactions.

I wasn't aware that Endless Sky uses many assets that weren't created specifically for it. I assumed you had created them and released them for free. As I parse the answer you quote, this revolves around games that were created mainly out of and around free assets instead of original creations. Since Endless Sky looks very well-polished and well-integrated, I believe it will be ok.

With your permissions I would still like to try and submit the game. Having it in a DRM-free system would make me happy.

**Comment by akien-mga** [comment]
>> "I used some free or borrowed assets from different game, is this a problem?"

> "Yes, this is an issue for us. We're strongly focused on more original content and free or borrowed assets mean that there's a very small chance we'll accept the game."

That sounds like an ~~anti-RPM Maker~~ anti-Asset Store-based Ubuntu games clause.
"Free" doesn't mean much in this context though, we don't know if they also refer to libre assets or only to using plain "royalty-free" assets.

> With your permissions I would still like to try and submit the game. Having it in a DRM-free system would make me happy.

Wouldn't it be better for the game creator to submit it? Or is it more a kind of wish that gamers can submit to GOG.com so that they consider contacting the dev?

**Comment by shezi** [comment]
>>    With your permissions I would still like to try and submit the game. Having it in a DRM-free system would make me happy.

> Wouldn't it be better for the game creator to submit it? Or is it more a kind of wish that gamers can submit to GOG.com so that they consider contacting the dev?

Yes, it would be better for them to submit it, but keeping bureaucracy away from the game creator so that they can focus in making games is also good.

**Comment by Andrettin** [comment]
There are some assets used by the game which come from OpenGameArt.com and other free sources, as per the "copyright" file:
https://github.com/endless-sky/endless-sky/blob/master/copyright

Hopefully this wouldn't be an issue, but from my understanding of the FAQ it would, and the consequence would be to disqualify nearly every open-source game. Thanks for pointing out that there are free games on GOG.com, I had missed that.

Yes, I agree that it would still be worth it to submit the game. But it should definitely be done by the rights-holder (Michael Zahniser).

**Comment by endless-sky** [comment]
The way I interpret the FAQ, what they are looking for is "original content" - so, they would reject a game if it's just a remix of graphics and content from other games that they already have, but I doubt they would reject a game just because it contains some sound files from Open Game Art.

I can try contacting them once the craziness from the Steam release has died down a bit.

**Comment by shezi** [comment]
That would be brilliant, thank you!

**Comment by geojak** [comment]
What's the state on this. Was gog contacted. If not is is possible to try another attempt?

**Comment by Zitchas** [comment]
This could be good. 
A) Maybe the gog system might be more flexible than the steam system about having multiple maintainers (aka, reducing the distribution bus-factor that we currently have with Steam).
B) Many people prefer GOG over Steam. This could expand Endless-Sky's exposure to other markets.

**Comment by Amazinite** [comment]
I was wondering a while back if it might make sense to wait for v1.0.0 before attempting to release on GOG.

**Comment by geojak** [comment]
I would not wait. There is no guaranty 1.0 is ever achieved. Better just push to gog now. Maybe it can attract more player and possibly some wants to help development.

**Comment by Zitchas** [comment]
Eh, I'm not sure. 1.0.0 still seems a really long ways off; and as a community project, more exposure is a good thing. If 1.0.0 looked imminent (within the next ten months, with a high degree of confidence), I'd say hold off on it. As it is, though, I'm more inclined to say let's aim for the 0.9.14 release being considered for a GOG submission.

**Comment by vpelss** [comment]
Endless Sky is one of the best games I have played in quite some time. The addiction factor is through the roof. This is GOG material. Make it happen :) Two of my faforite things. GOG & Endless Sky.

**Comment by ziproot** [comment]
If the issue is that the game isn't fully released, you could also try https://itch.io/. Many games are released there while they are still being made (see e.g. https://itch.io/t/13498/distributing-pre-release-builds-with-itchio or just google "itch.io pre release"). This is also drm free and you can host open source games there. Then you could move to GOG if/when 1.0 is released.

**Comment by vpelss** [comment]
Big pull from fans on GOG! https://www.gog.com/wishlist/games/endless_sky_space_exploration_free_and_open_source

**Comment by Hecter94** [comment]
I submitted a request to GoG. I figured there was no harm in reaching out and seeing what they said. 

That doesn't mean it will be added; I'm just going to try and talk with someone there and see if they'd even accept it if we were to try and submit it. 
If they say no, that's that; if they say they would, it would be up to our dev team, and I suppose MZ, if he has anything to say on the matter to discuss when/if we want to release on GoG.
